src/bin/flamegraph.rs

```diff
@@ -212,6 +212,13 @@ struct Opt {
     #[clap(long = "width", value_name = "UINT")]
     width: Option<usize>,
 
+    /// Filter out stacks that do not contain this symbol, when this symbol is in the stack,
+    /// truncate the call stack so that this is the bottom-most symbol.
```

Comment:
I couldn't quite follow this comment. Possibly because it's one long, complex, run-on sentence.